tabs/_common.py

```
@@ -1,4 +1,34 @@
 # No imports needed - this module provides pure data functions
+from shiny import ui
+
+def wrap_with_container(content):
```

Ruff (0.14.10)

Missing return type annotation for private function `wrap_with_container`

(ANN202)